dev-infrastructure/templates/region.bicep

```diff
@@ -66,5 +66,6 @@ module maestroInfra '../modules/maestro/maestro-infra.bicep' = {
     maxClientSessionsPerAuthName: maestroEventGridMaxClientSessionsPerAuthName
     maestroKeyVaultName: maestroKeyVaultName
     kvCertOfficerManagedIdentityName: maestroKeyVaultCertOfficerMSIName
+    publicNetworkAccess: 'Disabled'
```

Comment:
let's keep eventgrid public for now while also using the private endpoint

Reply:
Any particular reason?